### DISCUSSION TITLE
Can Copilot be used in a conversational way

### DISCUSSION THREAD

**Discussion by Aiswarya1794** [community/community#55720] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I have configured Github Copilot in my chatbot project and I am able to write code using copilot.
Can I use Copilot in a conversational way as same as chat gpt or it only supports programmers.

**Comment by Akash1134** [comment, marked as answer]
Hi @Aiswarya1794 ,

It seems that you are referring to the Copilot Chat feature, it is not designed to answer non-coding questions, and therefore its responses may not always be accurate or helpful in these contexts. If a user asks Copilot Chat a non-coding question, it may generate an answer that is irrelevant or nonsensical, or it may simply indicate that it is unable to provide a useful response.

Thanks!

**Comment by Aiswarya1794** [comment]
We have Bing chat available with 'AI-powered copilot for the web', so is it possible to use the copilot public data set for a chatbot
